llvm/test/Transforms/LoopIdiom/RISCV/byte-compare-index.ll

```diff
@@ -0,0 +1,1751 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --version 3
+; RUN: opt -passes=loop-idiom-vectorize -mtriple=riscv64-unknown-linux-gnu -loop-idiom-vectorize-style=predicated -mattr=+v -S < %s | FileCheck %s
```

Comment:
Can you add a version which uses the non-predicated style?  I'd like to know both work on RISCV.

Reply:
Just added.